lib/routes/account.js

```diff
@@ -509,13 +511,34 @@ module.exports = function (
           if (shouldSendNewDeviceLoginEmail) {
             // The response doesn't have to wait for this,
             // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
-              userAgent.call({
-                acceptLanguage: request.app.acceptLanguage,
-                timestamp: Date.now()
-              }, request.headers['user-agent'])
-            )
+            var location = null
+            var timeZone = null
+            geodb(ip)
```

Comment:
In [1] we specify some metrics events to log as part of this, e.g. `fxa.location.accuracy.unknown`.  It may be worth wrapping the `geodb(ip)` call in a little helper function that goes the lookup and logs the metrics, to keep that logic in a single place.  IIRC @vbudhram suggested the specifics of those metrics so he may be able to advise further here.

[1] https://github.com/mozilla/fxa/tree/master/features/proposed/FxA-99-improved-login-notification#metrics